util/tmpl.go

```diff
@@ -49,6 +49,10 @@ func MatchTmpl(license string, useSPDXIDs bool) (string, error) {
 		return tmplGPLThreeOrLater, nil
 	case "gpl-3.0-only":
 		return tmplGPLThreeOnly, nil
+	case "AGPL-3.0-or-later":
```

Comment:
Please use lowercase here, `"AGPL-3.0-or-later"` => `"agpl-3.0-or-later"`. 
`strings.ToLower` will convert the value of the license option entered by the user to lowercase for matching.
 https://github.com/B1NARY-GR0UP/nwa/blob/770bb5fa238d434251a26bb4a673b2cb37b9760b/util/tmpl.go#L42